modules/core/include/opencv2/core/hal/intrin_cpp.hpp

```diff
@@ -1088,6 +1089,23 @@ template<typename _Tp, int n> inline int v_signmask(const v_reg<_Tp, n>& a)
     return mask;
 }
 
+/** @brief Get first negative lane index
+
+Returned value is an index of first negative lane (undefined for input of all positive values)
+Example:
+@code{.cpp}
+v_int32x4 r; // set to {0, 0, -1, -1}
+int idx = v_heading_zeros(r); // idx = 2
+@endcode
+*/
+template <typename _Tp, int n> inline int v_find_negative(const v_reg<_Tp, n>& a)
```

Comment:
What is about these names?
- `v_find_non_zero()`
- `v_scan_non_zero()`
- `v_scan_forward()`
- `v_scan()`